### ISSUE TITLE
issue with pokemon let's go box editing

### ISSUE THREAD

**Issue by NinFanBoyFTW** [kwsch/PKHeX#2191]
Hi. I was editing my PC boxes in my let's go pikachu because I wanted to take some pokemon out, and put some pokemon in from a different folder on my computer. After I finished my editing, my party pokemon were moved around. In addition to this, now every time I go into my game, it says "play with _" instead of "play with pikachu" and the game crashes when I try to access my party. I tried to put my party where they were and changed everything to what I though would work but nothing is helping. Ill include the save file so you can maybe figure it out. This seems like it could be a major issue for other people eventually, as I literally only deleted a few pokemon and added in a bunch of new pokemon. Thanks

(kwsch edit: deleted save)

**Comment by kwsch** [comment]
This is likely due to PKHeX losing track of the partner/follower index, which is stored at 0x5A00.

![image](https://user-images.githubusercontent.com/6393368/49342595-4d358f00-f612-11e8-8748-04f122b279c1.png)

Try updating it from 0x3E9 (1001) to whatever slot your starter is in.

**Comment by NinFanBoyFTW** [comment]
Hey I was just looking at my save in a hex editor (this current one and a backup I have from right before editing) and the one from before editing has a hex value of 0E00 with the pikachu being placed in slot 15. I tried changing the current save to that and now it is working, thanks.

I am just wondering though, is there any way to have a setting in pkhex to "set as partner pokemon" when you right click on a pokemon. I think that there will be a lot of people who end up getting into this situation who will not be able to use a hex editor, plus im not really sure of how the numbers for the box slots are assigned so I couldn't do this without having a previous save. Also I have a weird "glitch" happen in my let's go eevee where it said "play with bellsprout" because I had moved around pokemon and once I moved eevee back to the party position that bellsprout was in it went back to normal, so if you were to add this feature (and have it as a general right click not just for partner pokemon) I think the worst that could happen is that people would have to re adjust their party if they chose a random pokemon as partner (not their game crashing) I hope you will consider it. Thanks
